src/Pretzel.Logic/Recipes/Recipe.cs

```diff
@@ -77,18 +75,17 @@ public void Create()
                         Tracing.Info("Wiki switch not valid with liquid templating engine");
                     CreateDirectories();
 
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"rss.xml"), Properties.Liquid.Rss);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"atom.xml"), Properties.Liquid.Atom);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"sitemap.xml"), Properties.Liquid.Sitemap);
-
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_layouts", "layout.html"), Properties.Liquid.Layout);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_layouts", "post.html"), Properties.Liquid.Post);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"index.html"), Properties.Liquid.Index);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"about.md"), Properties.Liquid.About);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_posts", string.Format("{0}-myfirstpost.md", DateTime.Today.ToString("yyyy-MM-dd"))), Properties.Liquid.FirstPost);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"css", "style.css"), Properties.Resources.Style);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_config.yml"), Properties.Liquid.Config);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_includes", "head.html"), Properties.Liquid.Head);
+                    CreateFile(@"Resources\Liquid\Rss.liquid", directory, @"rss.xml");
+                    CreateFile(@"Resources\Liquid\Atom.liquid", directory, @"atom.xml");
+                    CreateFile(@"Resources\Liquid\Sitemap.liquid", directory, @"sitemap.xml");
+                    CreateFile(@"Resources\Liquid\Layout.liquid", directory, @"_layouts", "layout.html");
+                    CreateFile(@"Resources\Liquid\Post.liquid", directory, @"_layouts", "post.html");
+                    CreateFile(@"Resources\Liquid\Index.liquid", directory, @"index.html");
+                    CreateFile(@"Resources\Liquid\About.liquid", directory, @"about.md");
+                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_posts", string.Format("{0}-myfirstpost.md", DateTime.Today.ToString("yyyy-MM-dd"))), GetResourceString(@"Resources\Liquid\FirstPost.liquid"));
```

Comment:
This one can't use `CreateFile`?

Reply:
Totally overlooked that one, was irritated by the `string.format`

Reply:
Should be fixed now.